hammr/utils/migration_utils.py

```diff
@@ -29,3 +36,83 @@ def migration_table(migrations):
         table.add_row([migration.dbId, migration.name, status])
 
     return table
+
+
+def check_account_from_platform(accounts, credAccountName, targetPlatformName):
+    if accounts is None or len(accounts) == 0:
+        printer.out(
+            "No accounts available on the plateform.\n You can use the command 'hammr account create' to create an account.",
+            printer.ERROR)
+        return
+    else:
+        for account in accounts:
+            if account.name == credAccountName:
+                if account.targetPlatform.name == targetPlatformName:
+                    return account
+        return
```

Comment:
shouldn't we raise an exception to be coherent with other functions?

Reply:
done